task/01-strings-tasks.js

```diff
@@ -201,7 +201,39 @@ function extractEmails(str) {
  *
  */
 function getRectangleString(width, height) {
-    throw new Error('Not implemented');
+    var rect = '';
+
+    for (var i = 0; i < height; i++) {
+        if (i === 0) {
+            rect += '┌';
+        }
+        else if (i !== 0 && i !== height - 1) {
+            rect += '│';
+        }
+        else {
+            rect += '└';
+        }
+
+        for (var j = 0; j < width - 2; j++) {
+            if (i === 0 || i === height - 1) {
+                rect += '─';
+            }
+            else {
+                rect += ' ';
+            }
+        }
+
+        if (i === 0) {
+            rect += '┐\n';
+        }
+        else if (i !== 0 && i !== height - 1) {
+            rect += '│\n';
+        }
+        else {
+            rect += '┘\n';
+        }
+    }
+    return rect;
 }
```

Comment:
`var` is deprecated. It has no sense to use for browsers code even currently.

Reply:
ok i got it